### DISCUSSION TITLE
What are these things called in the game scripts?

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#94] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51859157/209452281-957af0da-37e6-4713-88e0-89dee7954a0b.png)
What do the scrpting files call these stacks in the tech hub where we get the flight recorder?  
I am speeding up some bits in lieu of skipping or bypassing things.  
I have no idea what they're actually called to find their animations. In a real library they'd be a movable stack or something.  
I cannot find where the objects are located in the game files though.   Should be linked to buttons, but there are no Tiny Button 
objects other than for the door locks.  I'm stumped.

**Comment by ghramsey** [comment, marked as answer]
I found it! 
Archetypes\Misc_Subarchetypes\use_recordstorage 
ANIpush_button is the animation container.  
Settings PlaySpeed to something like 10 or 20 will make them go much faster.

*edit* I take it back.  The use_recordstorage only speeds up the button press not the record storage things' animation.
It's faster but I'd love both of them to be sped up.


![image](https://user-images.githubusercontent.com/51859157/210671767-297515ad-fade-4a0c-bc2a-5256e6e402cd.png)
Found it!  Archetypes>animatedrecordstorage

**Reply by MattFiler** [reply]
Hmm, it's probably a CAGEAnimation - worth having a look through those. You should be able to find the trigger for it off of the USE button press, as that'd be activating it.

**Reply by ghramsey** [reply]
I've been through all of the cage animation and none seem to be applicable.  
As I edited I found the button, but speeding up the button didn't have any options for the 'record storage' stacks.
I'll keep searching.

**Comment by ghramsey** [comment]
I did actually hit on the place to speed up these stacks. In the screen shot there are speed settngs. 
[RotatateAtSpeed_1] and [RotateAtSpeed_2]. I believe they have a y speed factor.
